test/boost/cql_query_test.cc

```diff
@@ -125,6 +125,34 @@ SEASTAR_TEST_CASE(test_create_twcs_table_no_ttl) {
     });
 }
 
+SEASTAR_TEST_CASE(test_twcs_max_window) {
+    return do_with_cql_env_thread([](cql_test_env& e) {
+        // Create a TWCS table with no defined TTL
+        e.execute_cql("CREATE TABLE tbl (a int, b int, PRIMARY KEY (a)) WITH "
```

Comment:
I don't think you need this specific check here - I think you already have it in the other test in the previous patch (test_create_twcs_table_no_ttl) that deals with no TTL at all.